gammacat/dataset.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+from .utils import load_yaml, write_yaml
+from gammapy.catalog.gammacat import GammaCatResource
+
+__all__ = [
+    'DataSet',
+]
+
+log = logging.getLogger(__name__)
+
+class DataSet:
```

Comment:
What is this class? Add a one-line docstring at least.

And please remove the one-line methods like source_id that just re-expose something from data. They don't help and just add more code to read and maintain.